### PR TITLE
Build the Swift package

### PR DESCRIPTION
**Pull request by hamermike**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: Build Swift Package
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
```
Comment (jmulford-bw) [inline .github/workflows/build.yml:8]: So this is both a PR check and merge condition?

Comment (hamermike) [reply .github/workflows/build.yml:8]: Yeah, I figured might as well verify on both before and after the merge it's still all good (only takes a minute to build).

Comment (jmulford-bw) [reply .github/workflows/build.yml:8]: Ok works for me

**Review by jmulford-bw — APPROVED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (hamermike) [reply .github/workflows/build.yml:8]: It's probably overkill by a lot at the moment but may be more useful down the road.

**Pull request merged**